src/components/cms/BrowseSkill/skillLoader.js

```diff
@@ -14,9 +14,23 @@ const ContentContainer = styled.div`
   border: 0.5px solid #eaeded;
 `;
 
+const loaderLimit = () => {
+  let limit = 2;
+  if (window.innerWidth >= 1680) {
```

Comment:
Store window.innerWidth in a variable and then use that variable.

Reply:
Sure :+1:

Reply:
It'll take some time